apisix/core/etcd.lua

```diff
@@ -507,7 +507,7 @@ function _M.atomic_set(key, value, ttl, mod_revision)
         key = key,
         value = value,
     }
-    res.status = 201
+    res.status = 200
```

Comment:
Is the default status already 200?

Reply:
IMHO yes. Only the PATCH method uses this function now.

Reply:
So we don't need to reassign it at all

Reply:
The res.status is assigned here and is only called by `PATCH`. That seems reasonable, doesn't it?